crates/bevy_gltf/src/loader.rs

```diff
@@ -142,6 +146,23 @@ async fn load_gltf<'a, 'b>(
                 mesh.set_attribute(Mesh::ATTRIBUTE_UV_0, vertex_attribute);
             }
 
+            if let Some(vertex_attribute) = reader.read_joints(0).map(|v| {
```

Comment:
The new code in this function doesn't seem to be feature gated.

Reply:
This code is in `bevy_gltf` crate, and it depends on the `bevy_animation_rig`. Making `bevy_animation_rig` optional in this crate is kinda hard as `bevy_gltf` is also optional for `bevy`. I don't know how to deal with this. Do you have any suggestion?